.changeset/twelve-toes-search.md

```diff
@@ -0,0 +1,5 @@
+---
+"pie-docs": minor
+---
+
+[Changed] - Removed /overview urls from foundations pages and use overview content as the index.html of each directory. I.e. /foundations/colour for the overview page instead of /foundations/overview/colour. Also now uses the 404 rather than coming soon partial for empty pages as the coming soon was misleading and often not true.
```

Comment:
Nit
```suggestion
[Changed] - Removed /overview urls from foundations pages and use overview content as the index.html of each directory. I.e. /foundations/colour for the overview page instead of /foundations/colour/overview. Also now uses the 404 rather than coming soon partial for empty pages as the coming soon was misleading and often not true.
```